akka-docs/src/main/paradox/stream/operators/Source-or-Flow/conflateWithSeed.md

```diff
@@ -18,6 +18,25 @@ Allow for a slower downstream by passing incoming elements and a summary into an
 is backpressure. When backpressure starts or there is no backpressure element is passed into a `seed` function to
 transform it to the summary type.
 
+## Example
+
+Scala
+:   @@snip [SourceOrFlow.scala](/akka-docs/src/test/scala/docs/stream/operators/SourceOrFlow.scala) { #conflateWithSeed }
+
+Java
+:   @@snip [SourceOrFlow.java](/akka-docs/src/test/java/jdocs/stream/operators/SourceOrFlow.java) { #conflateWithSeed-type #conflateWithSeed }
+
+
+If downstream is slower, the "seed" function is called which is able to change the type of the to be conflated
+elements if needed (it can also be an identity function, in which case this `conflateWithSeed` is equivalent to 
+a plain `conflate`). Next, the conflating function is applied while back-pressure from the downstream is upheld,
+such that the upstream can produce elements at an rate independent of the downstream.
```

Comment:
Skip first comma. "while there is backpressure" instead of the "upheld" phrasing"?